lib/tbot_web/messenger/tbot_messenger_task.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Tbot.MessengerOutputTask do
+  @moduledoc """
+  Task responsible for building the Messenger resquest body and
+  sending it back to Messenger's user.
+  """
+
+  alias Tbot.MessengerOutput, as: MessengerOutput
+
+  def respond_messenger(parsed_entry) do
+    # NOTE: Not sure if "async_nolink" is the best supervisor option
+    Task.Supervisor.async_nolink Tbot.TaskSupervisor, fn ->
```

Comment:
For this purpose `async_nolink` seems a good option. Take a look in the docs about `async` and `async_nolink` for more details: https://github.com/elixir-lang/elixir/blob/v1.5.3/lib/elixir/lib/task/supervisor.ex#L68